mimic/model/clb_objects.py

```diff
@@ -10,11 +10,23 @@
 from mimic.canned_responses.loadbalancer import (load_balancer_example,
                                                  _verify_and_update_lb_state,
                                                  _lb_without_tenant,
-                                                 _prep_for_list,
                                                  _format_nodes_on_lb,
                                                  _delete_node)
 
 
+def _prep_for_list(lb_list):
```

Comment:
If this is being migrated instead of imported, we can probably delete it in `canned_responses.loadbalancer`, since nothing else uses it there.  Leaving it will probably cause coverage to complain, since that code will no longer be exercised.